src/main/java/com/efficio/fieldbook/web/fieldmap/controller/FieldmapController.java

```diff
@@ -139,6 +135,15 @@ public Map<String, String> determineFieldMapNavigation(@PathVariable final Strin
 		return result;
 	}
 
+	private List<Integer> getTrialIds(final String ids) {
```

Comment:
@cheryleigh can be done  with lambda or Arrays. 
check https://stackoverflow.com/questions/7488643/how-to-convert-comma-separated-string-to-list